homeassistant/components/tmb/sensor.py

```diff
@@ -0,0 +1,124 @@
+"""Support for TMB (Transports Metropolitans de Barcelona) Barcelona public transport."""
+from datetime import timedelta
+import logging
+import requests
```

Comment:
Only import `HTTPError` from requests instead of importing the entire module.

Reply:
Thanks, all comments addressed